migrations/9_create_user_emails_array.js

```diff
@@ -0,0 +1,49 @@
+const Settings = require('settings-sharelatex')
+const mongojs = require('mongojs')
+const db = mongojs(Settings.mongo.url, ['users'])
+const async = require('async')
+
+const handleExit = () => console.log('Got signal.  Shutting down.')
```

Comment:
Looks like `handleExit` prints a log line, but doesn't exit?

Reply:
Good catch!

Reply:
Wait, checked other migrations, they're just logging a `SIGINT`, `SIGHUP` exit.

Reply:
:thinking: well, if it's good enough for the other migrations then it's good enough for us I guess, but I can't see the point of this except to eat the signals instead of responding to them. In which case, the message is incorrect as really it's `Got signal. Ignoring.`

You're right though, it's not going to take very long so we shouldn't lose sleep over it.